handler.go

```diff
@@ -145,6 +145,16 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, u *User) {
 		return
 	}
 
+	// If authentication is enabled and there are no users yet, make sure
+	// the first statement is creating a new cluster admin.
+	if h.AuthenticationEnabled && h.server.UsersLen() == 0 {
+		stmt, ok := query.Statements[0].(*influxql.CreateUserStatement)
+		if !ok || stmt.Privilege == nil || *stmt.Privilege != influxql.AllPrivileges {
+			h.error(w, "must create cluster admin", http.StatusBadRequest)
```

Comment:
@pauldix and I spoke about this last week on Slack. He does want that 401 in there too, so the entire set of codes we might send back are 200, 400, 401, and 500. I'll let him confirm though.